README.md

```diff
@@ -1,26 +1,26 @@
-# Go Mod Cloud Native Buildpack
+# Go Mod Vendor Cloud Native Buildpack
 
-The Go Mod CNB builds a Go application binary, using the [`go
+The Go Mod Vendor CNB builds a Go application binary, using the [`go
 mod`](https://golang.org/cmd/go/#hdr-Modules__module_versions__and_more)
 functionality provided by the Go Compiler CNB to package dependencies.
 
 ## Integration
 
-The Go Mod CNB provides go-mod as a dependency. Downstream
-buildpacks can require the node dependency by generating a [Build Plan
+The Go Mod Vendor CNB provides go-mod-vendor as a dependency. Downstream
+buildpacks can require the go-mod-vendor dependency by generating a [Build Plan
 TOML](https://github.com/buildpacks/spec/blob/master/buildpack.md#build-plan-toml)
 file that looks like the following:
 
 ```toml
 [[requires]]
 
-  # The name of the Go Mod dependency is "go-mod". This value is
+  # The name of the Go Mod dependency is "go-mod-vendor". This value is
   # considered part of the public API for the buildpack and will not change
   # without a plan for deprecation.
-  name = "go-mod"
+  name = "go-mod-vendor"
```

Comment:
As the comment above this field mentions, this is part of the public API and will not be changed without a plan for deprecation. Do we have a plan? Or should this remain how it was?

Reply:
We aren't changing this from `go-mod`, the README change was an accidental find and replace, thanks for catching that!